2013/208.vtt

```diff
@@ -674,15 +643,76 @@ with you this morning to go
 into some detail about iOS
 7 and some of the new apps
 
-<HTML><HEAD>
-<TITLE>Request Timeout</TITLE>
-</HEAD><BODY>
-<H1>Request Timeout</H1>
-The server timed out while waiting for the browser's request.<P>
-Reference&#32;&#35;2&#46;d5524817&#46;1373789674&#46;0
-</BODY></HTML>
-WEBVTT
-X-TIMESTAMP-MAP=MPEGTS:181083,LOCAL:00:00:00.000
+00:11:03.406 --> 00:11:04.636 A:middle
+like Mail and Calendar.
```

Comment:
When the previously-used ruby script encounters an HTTP error on fetching an individual VTT playlist file, it concatenates the HTML error response. Below is the missing caption content that replaces the error message. There are several of these corrections. The tool will fail the entire fetch of the session's transcript if such an error is encountered.